.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+
+on:
+  push:
+    branches: 
+    - 'maintenance/0.2.18'
+  pull_request:
+    branches: 
+    - 'maintenance/0.2.18'
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          python-version: 2.7
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: sudo apt-get install poppler-utils
+
+      - name: Set up Python 2.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+          pip install -e .[docs,tests]
+
+      - name: Show python dependencies
+        run: |
+          pip freeze
+
+      - name: Run tests
+        run: |
+          ./run-tests.sh
```

Comment:
I think you also need to do the release, like in https://github.com/inspirehep/refextract/blob/8cdb6f1d37b140f3b9bd05b06b52aabaf1463e0c/.github/workflows/build-and-release.yml#L41-L53